src/bootstrap.ts

```diff
@@ -156,8 +157,8 @@ const setupRxNormConnection = (conf: RxNormDbConfig) => {
 
 async function pingRxNorm(pool: Pool) {
 	try {
-		const query = promisify(pool.query).bind(pool);
-		await query('select 1');
+		const connection = await pool.getConnection();
+		await connection.query('select 1');
```

Comment:
here `getConnection` is used before querying, later in `MysqlRxNormService` class `query` is used directly from `pool`. Might be good to keep it consistent. What is the difference between these two approaches?

Reply:
edit: actually, pool.query is a source of connection errors in our tests
So I'm going to use getConnection with a manual release